### ISSUE TITLE
converting image to video not working

### ISSUE THREAD

**Issue by panand1992** [fluent-ffmpeg/node-fluent-ffmpeg#596]
I am trying to run the one of the example in the examples folder 
image2video.js and every time it fails with 

> Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height

I have ffmpeg and ffprobe added to my path. FFmpeg details are below

ffmpeg version 3.0.2 Copyright (c) 2000-2016 the FFmpeg developers
  built with Apple LLVM version 7.3.0 (clang-703.0.29)
  configuration: --prefix=/usr/local/Cellar/ffmpeg/3.0.2 --enable-shared --enable-pthreads --enable-gpl --enable-version3 --enable-hardcoded-tables --enable-avresample --cc=clang --host-cflags= --host-ldflags= --enable-opencl --enable-libx264 --enable-libmp3lame --enable-libxvid --enable-vda
  libavutil      55. 17.103 / 55. 17.103
  libavcodec     57. 24.102 / 57. 24.102
  libavformat    57. 25.100 / 57. 25.100
  libavdevice    57.  0.101 / 57.  0.101
  libavfilter     6. 31.100 /  6. 31.100
  libavresample   3.  0.  0 /  3.  0.  0
  libswscale      4.  0.100 /  4.  0.100
  libswresample   2.  0.101 /  2.  0.101
  libpostproc    54.  0.100 / 54.  0.100

Please tell me if I am doing anything wrong .

**Comment by abhikmitra** [comment]
+1 facing the same problem

**Comment by pirmax** [comment]
+1 Same problem :/

**Comment by njoyard** [comment]
Yeah the examples are sometimes outdated unfortunately. You should determine what ffmpeg command line you want to run and then use the docs to convert it to a fluent-ffmpeg call.
